lighthouse-core/report/html/renderer/details-renderer.js

```diff
@@ -460,6 +467,102 @@ class DetailsRenderer {
     if (item.selector) element.setAttribute('data-selector', item.selector);
     if (item.snippet) element.setAttribute('data-snippet', item.snippet);
 
+    if (!item.boundingRect || !this._fullPageScreenshot) {
+      return element;
+    }
+
+    const fullPageScreenshot = this._fullPageScreenshot;
+
+    const elementScreenshot = ElementScreenshotRenderer.render(
+      this._dom,
+      this._templateContext,
+      item,
+      fullPageScreenshot,
+      {width: 147, height: 100}
+    );
+
+    element.prepend(elementScreenshot);
+
+    // TODO: should this go in report ui features?
+    elementScreenshot.addEventListener('click', () => {
+      const overlay = this._dom.createElement('div');
+      // @ts-ignore: style is not read-only.
+      overlay.style = `
+        position: fixed;
+        top: 0;
+        left: 0;
+        width: 100vw;
+        height: 100vh;
+        z-index: 1;
+        background: rgba(0, 0, 0, 0.7);
+        display: flex;
+        align-items: center;
+        justify-content: center;
+      `;
+      overlay.appendChild(ElementScreenshotRenderer.render(
+        this._dom,
+        this._templateContext,
+        item,
+        fullPageScreenshot,
+        {
+          // TODO: should this be documentElement width?
+          width: window.innerWidth * 0.75,
+          height: window.innerHeight * 0.75,
+        }
+      ));
+      document.body.appendChild(overlay);
+      overlay.addEventListener('click', () => {
+        overlay.remove();
+      });
+    });
+
+    // element.addEventListener('click', () => {
+    //   console.log('click', elementScreenshot);
+    //   // for mobile
+    //   // todo: figure out if needs any other handlers like click to show, maybe disable hover?
+    //   if (elementScreenshot) {
+    //     console.log('removing');
```

Comment:
are these still necessary or just being worked on?

Reply:
we still need to discuss how we want the image to show in the report on click, @mattzeunert had this code commented out by the time I took over. keepin it around pending discussion